lib/src/pub.dart

```diff
@@ -87,6 +108,19 @@ class Pubspec {
       this._contents,
       this._yamlMap);
 
+  static Pubspec init() {
```

Comment:
Make this return a Future<Pubspec> and use the async git methods.  Reason is I'm considering calling the github API to get the description later on, which would have to be async.